.pylintrc

```diff
@@ -34,8 +34,7 @@ disable=all
 # Enable the message, report, category or checker with the given id(s). You can
 # either give multiple identifier separated by comma (,) or put this option
 # multiple time. See also the "--disable" option for examples.
-enable=import-error,
```

Comment:
if pylint runs in an isolated virtualenv, then it won't be able to import third-party modules. However, invalid imports would already raise an error during the pytest jobs, so this check is redundant and can be removed